pgtransaction/transaction.py

```diff
@@ -104,8 +106,11 @@ def atomic(
     savepoint=True,
     durable=False,
     isolation_level=None,
-    retry=0,
+    retry=None,
 ):
+    if retry is None:
+        retry = config.retry()
```

Comment:
I thought about it some more and ultimately thought it might lead to situations where people accidentally have infinite retry loops and overload their DB, especially since we allow configuring the exceptions handled